docs/tutorials/02_create_your_first_app/06_add-frontend.md

```diff
@@ -77,6 +76,8 @@ const pollResults = {
 
 //Load the Simple Poll's question from the backend when the app loads
 document.addEventListener('DOMContentLoaded', async (e) => {
+  e.preventDefault();
```

Comment:
Does this do anything? I was under the impression that DOMContentLoaded isn't cancelable.